spec/features/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+feature 'user stuff' do
+
+  scenario 'I can sign up as a new user' do
+      user = build :user
+      expect { sign_up(user) }.to change(User, :count).by(1)
+      expect(page).to have_content('Welcome, foo@bar.com')
+      expect(User.first.email).to eq('foo@bar.com')
+    end
```

Comment:
Lint/BlockAlignment: `end` at 8, 4 is not aligned with `scenario 'I can sign up as a new user' do` at 3, 2